trestle/core/commands/split.py

```diff
@@ -129,7 +201,9 @@ def split_model_at_path_chain(
         cur_path_index: int,
         split_plan: Plan,
         strip_root: bool,
-        root_file_name: str = ''
+        root_file_name: str,
+        aliases_to_strip: Dict[str, AliasTracker],
+        last_one: bool = True
     ) -> int:
         """Recursively split the model at the provided chain of element paths.
```

Comment:
Check the documentation strings.

Reply:
Repeat of above?  Have added doc strings.